.github/workflows/node-docker.yml

```diff
@@ -79,31 +101,48 @@ jobs:
           username: subquerynetwork
           password: ${{ secrets.SQ_DOCKERHUB_TOKEN }}
 
+      - run: yarn
+
+      - name: build
+        run: yarn build
+
       ## node
       - name: Get updated node version
         id: get-node-version
         run: |
           sh .github/workflows/scripts/nodeVersion.sh
 
+      - name: Set execute permissions for the script
+        run: chmod +x .github/workflows/scripts/replace-versions.sh
```

Comment:
I think if you run this locally then commit then this doesn't need to be done in CI